README.md

```diff
@@ -2,11 +2,13 @@
 ENGI 7854 Image Processing and Analysis Project
 
 Problem Statement:
+
 Determining the result of a Go game by manually counting the territory/points can be tedious and error-prone, 
 especially after an intense long match because the resulting Go board can be very complicated. 
 One could use an automated machine vision system to facilitate this process.
 
 System Description:
+
 The system will take a picture of a Go board after a match as input 
 and give the result of the game as following quickly (less than 15s) and with no error:
 •	Black Total Points
```

Comment:
These bullet points should be on their own lines. Adding <br /> to the end of each point should fix this.